config/config.py

```diff
@@ -47,7 +49,11 @@ def anonymous_tracking_enabled(self) -> bool:
 
     @property
     def slack_notification_webhook(self) -> Union[str, None]:
-        return self.config_dict.get(self.SLACK_NOTIFICATION_WEBHOOK)
+        return self.config_dict.get(self.SLACK).get(self.NOTIFICATION_WEBHOOK)
+
+    @property
+    def is_slack_workflow(self) -> bool:
+        return True if self.config_dict.get(self.SLACK).get(self.WORKFLOWS) else False 
```

Comment:
Same comment as above

Reply:
[adding null check for slack workflow config](https://github.com/elementary-data/elementary/pull/54/commits/49f5cd4f2790bbfc433c636f9f714ea501279aa9)